config/services.yaml

```diff
@@ -62,8 +66,6 @@ services:
         tags:
             - { name: monolog.processor }
 
-    # add more service definitions when explicit configuration is needed
-    # please note that last definitions always *replace* previous ones
```

Comment:
Why remove this comment... we have before fallen into this issue.

Reply:
Not deleted, just moved to improve structure of file